starter-code/scripts/models/article.js

```diff
@@ -56,7 +60,16 @@ Article.fetchAll = function() {
      1. Load our json data,
      2. Store that data in localStorage so we can skip the server call next time.
      3. And then render the index page. */
+    $.getJSON('../data/hackerIpsum.json',function(data){
+      localStorage.hackerIpsum = JSON.stringify(data);
+      Article.loadAll(data);
+      articleView.renderIndexPage();
+    });
+
+    // localStorage.hackerIpsum = JSON.stringify(newData);
+
   }
```

Comment:
Since you're doing `Article.loadAll()` and `articleView.renderIndexPage()` whether or not there's data in local storage, you should probably just remove it from the `if...else...` block and have those two functions run regardless of whether or not there's data in local storage.